corehq/motech/models.py

```diff
@@ -289,5 +290,5 @@ def log(level: int, log_entry: RequestLogEntry):
             request_error=log_entry.error,
             response_status=log_entry.response_status,
             response_headers=log_entry.response_headers,
-            response_body=as_text(log_entry.response_body),
+            response_body=as_text(log_entry.response_body)[:ONE_MB],
```

Comment:
Great find. 1MB is certainly better than no limit, but 1MB of text is still an enormous amount, enough to fill a few pages. Hard to believe this will be useful to people. I'm fine with starting here and lowering it later after more thought.

Reply:
7200 lines of compact text apparently: https://gist.github.com/khaykov/a6105154becce4c0530da38e723c2330 🔠